README.md

```diff
@@ -93,15 +98,11 @@ A DID Document ("DIDDoc") associated with an earth DID is a set of data describi
 
 ### Linked Resources
 
-The `linkedResource` property provides a privacy-enabled way to attach digital resources to an on-chain asset. This is an optional property which may contain one or more resource descriptors in array. This property provides the metadata required for accessing and using the specified resource, such as the type of resource, a proof to verify the resource, and a service endpoint for requesting and retrieving the resource.
+The `linkedResource` property provides a privacy-enabled way to attach digital resources to an on-chain asset. This is an optional property which may contain one or more resource descriptors in an array. This property provides the metadata required for accessing and using the specified resource, such as the type of resource, a proof to verify the resource, and a service endpoint for requesting and retrieving the resource.
 
 Resources may be provided in-line or by secure reference. Inline resources are appropriate only for use cases that need to directly include the resource in the IID Document. In many cases, this is a privacy problem. However, for some use cases, resources must be specified for on-chain execution, which justifies the added bytes and potential disclosure risk. The resource descriptor provides for a flexible representation of various mime types, compression, and encoding, as required for the use.
 
-This approach allows token owners to manage privacy in three key ways:
-
-1.  Avoids posting potentially sensitive information on-chain in an unavoidably public and irrevocable manner.
-2.  Provides a service endpoint that can apply appropriate privacy and security checks before revealing information.
-3.  The hashgraph resource descriptor type obscures not only the content of the linked resource, but also the quantity of resource objects.
```

Comment:
@jandrieu do you want me to put this paragraph back here then? All I did was refactor this out into a privacy considerations section to start with.

Reply:
Nope. This can go.